core/platform-api-client-gwt/che-core-client-gwt-git/src/main/java/org/eclipse/che/api/git/gwt/client/GitServiceClientImpl.java

```diff
@@ -193,7 +194,7 @@ public void apply(MessageBus arg) throws OperationException {
     /** {@inheritDoc} */
     @Override
     public void statusText(String workspaceId, ProjectConfigDto project, StatusFormat format, AsyncRequestCallback<String> callback) {
-        String url = extPath + "/git/" + workspaceId + STATUS;
+        String url = appContext.getDevMachine().getWsAgentBaseUrl() + "/git/" + workspaceId + STATUS;
```

Comment:
It would better to rework GitServiceClient to accept DevMachine like https://github.com/eclipse/che/pull/1052/files#diff-2c60681c9574346a780995cc4690a051R41 or use only dev machine like https://github.com/eclipse/che/pull/1052/files#diff-cccdfa39d51dbd2ff8cc1b806160fe07R67